recipes/mdsrv/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "MDsrv" %}
+{% set version = "0.3.5" %}
+{% set sha256 = "3dd218ec0d62cf47061699b8ab33715d6875fad824091f2ab37e546467fef0c3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install 
+
+requirements:
+  build:
+    - python
+    - cython
+    - numpy >=1.11
+    - msinttypes  # [win and py2k]
+    - setuptools
+    - flask
+    - scipy
+    - zlib
+    - mdtraj
+    - mdanalysis  # [not win and py2k]
+
+  run:
+    - python
+    - setuptools
+    - numpy >=1.11
+    - flask
+    - mdtraj
+    - mdanalysis  # [not win and py2k]
+
+test:
+  imports:
+    - mdsrv
+
+about:
+  home: https://github.com/arose/mdsrv
+  license: MIT
+  summary: "Simple server to visualize remote trajectories."
+  description: |
+    MDsrv is a simple server that enables remote access to coordinate trajectories from molecular dynamics simulations. It can be used together with the NGL Viewer (http://github.com/arose/ngl) to interactively view trajectories of molecular complexes in a web-browser, either within a local network or from anywhere over the internet.
+  doc_url: https://nglviewer.org/mdsrv
+  dev_url: https://github.com/arose/mdsrv
+
+extra:
+  recipe-maintainers:
+    - j0kaso
+    - arose
```

Comment:
@arose -- are you ok being added as a co-maintainer of the `MDsrv` package on conda-forge? Your name was added by @j0kaso. Being a co-maintainer would give you write permission to the github repository allowing you to update the package on conda-forge.